### PR TITLE
Week 2 proj 2

### PR DESCRIPTION
**Pull request by cyber-mary**
Nav bar has link to contact page. Contact page has form to leave name and message, then redirects to thank you page.

### CODE REVIEW

**Review by nelango — COMMENTED** [review]
This PR includes changes from prior PR - https://github.com/cyber-mary/sps-portfolio/pull/3. Please merge https://github.com/cyber-mary/sps-portfolio/pull/3 and then rebase this PR to exclude prior changes.

**Comment by cyber-mary** [comment]
@go-nelango changes were made and PR is now ready for review

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 1]
```diff
@@ -1,18 +1,33 @@
...
 public class HelloWorldServlet extends HttpServlet {
+    private final List messages = new ArrayList<>(Arrays.asList("Bad Bunny", "SZA", "Rico Nasty", "Frank Ocean", "Joyce Manor"));
 
+    
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+  public void doGet(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
+
```
Comment (nelango) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java:21]: Nit: Remove newline

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 2]
```diff
@@ -1,18 +1,33 @@
...
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
+    private final List messages = new ArrayList<>(Arrays.asList("Bad Bunny", "SZA", "Rico Nasty", "Frank Ocean", "Joyce Manor"));
 
+
```
Comment (nelango) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java:18]: Nit: Remove newline

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 3]
```diff
@@ -1,18 +1,33 @@
...
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+  public void doGet(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
+
+     String json = convertToJsonUsingGson(messages);
+
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+  //Converts elements to JSON using Gson Library
```
Comment (nelango) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java:27]: Nit: Add newline after function

**Review by nelango — APPROVED** [review]
Looks good to me!

**Pull request merged**